corehq/apps/export/export.py

```diff
@@ -402,37 +395,10 @@ def _time_in_milliseconds():
     return int(time.time() * 1000)
 
 
-def _record_datadog_export_compute_rows(duration, doc_bytes, n_rows, tags):
-    __record_datadog_export(duration, doc_bytes, n_rows, 'commcare.compute_export_rows_duration', tags)
-
-
-def _record_datadog_export_write_rows(duration, doc_bytes, n_rows, tags):
-    __record_datadog_export(duration, doc_bytes, n_rows, 'commcare.write_export_rows_duration', tags)
-
-
 def _record_datadog_export_duration(duration, doc_bytes, n_rows, tags):
-    __record_datadog_export(duration, doc_bytes, n_rows, 'commcare.export_duration', tags)
-
-
-def __record_datadog_export(duration, doc_bytes, n_rows, metric, tags):
-    # deprecated: datadog histograms used this way are usually meaningless
-    # because they are aggregated on a 10s window (so unless you're constantly
-    # processing at least 100 measurements per 10s window then the percentile
-    # will approximate the average). Also more nuanced reasons. See
-    # https://confluence.dimagi.com/x/4ArDAQ#TroubleshootingPasteboard/HQchatdumpingground-Datadog
-    datadog_histogram(metric, duration, tags=tags)
-    if doc_bytes:
-        datadog_histogram(
-            '{}_normalized_by_size'.format(metric),
-            duration // doc_bytes,
-            tags=tags,
-        )
-    if n_rows:
-        datadog_histogram(
-            '{}_normalized_by_rows'.format(metric),
-            duration // n_rows,
-            tags=tags,
-        )
+    metrics_counter('commcare.export.duration', duration, tags=tags)
+    metrics_counter('commcare.export.rows', n_rows, tags=tags)
+    metrics_counter('commcare.export.bytes', doc_bytes, tags=tags)
```

Comment:
replaced the normalized metrics with the value counts. Any additional math can be done in DD

Reply:
👍